documentation/Set-PnPTenant.md

```diff
@@ -833,8 +833,8 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -PreventExternalUsersFromResharing
-Prevents external users from resharing files, folders, and sites that they do not own.
+### -PreventExternalUsersFromReSharing
+Prevents external users from reSharing files, folders, and sites that they do not own.
```

Comment:
The description text needs to be `resharing`.

Reply:
Oh, ups :)